README.md

```diff
@@ -205,17 +200,16 @@ Example:
 
 > Mention all of the collaborators of this project.
 
-👤 **Author1**
+👤 **Claudia Rojas**
 
-- GitHub: [@githubhandle](https://github.com/githubhandle)
+- GitHub: [@ClaudiaRojasSoto](https://github.com/ClaudiaRojasSoto)
 - Twitter: [@twitterhandle](https://twitter.com/twitterhandle)
-- LinkedIn: [LinkedIn](https://linkedin.com/in/linkedinhandle)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/claudia-soto-260504208/)
 
 👤 **Author2**
 
 - GitHub: [@githubhandle](https://github.com/githubhandle)
-- Twitter: [@twitterhandle](https://twitter.com/twitterhandle)
-- LinkedIn: [LinkedIn](https://linkedin.com/in/linkedinhandle)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/claudia-soto-260504208/)
```

Comment:
- Please remove the second author from this project, since this is a solo project and you worked alone, You should have links to only your social handles. In your subsequent Microverse project, You can include other authors when or if it is a pair-programming activity 😄